### PR TITLE
Landing Page Restyle

### PR DESCRIPTION
**Pull request by samoslag**
Restyled landing page with responsive support for smaller screens.

### CODE REVIEW

File: frontend/components/LastTransfers.vue [hunk 1]
```diff
@@ -1,65 +1,59 @@
...
+        :to="`/blocks`"
+        title="Click to see last transfers"
+      >
+        Last transfers
+      </nuxt-link>
     </div>
+
+    <Table>
+      <Row v-for="(item, index) in transfers" :key="'item-' + index">
+        <Cell label="Block" :link="`/transfer/${item.hash}`">{{
```
Comment (mariopino) [inline frontend/components/LastTransfers.vue]: Label should be `Hash` or `Id`

Comment (samoslag) [reply frontend/components/LastTransfers.vue]: Fixed

File: frontend/components/LastEvents.vue [hunk 2]
```diff
@@ -1,32 +1,31 @@
...
-          <p class="mb-0">
-            {{ data.item.section }} ➡
-            {{ data.item.method }}
-          </p>
-        </template>
-      </b-table>
+    <div class="headline">
+      <nuxt-link
+        v-b-tooltip.hover
+        :to="`/blocks`"
```
Comment (mariopino) [inline frontend/components/LastEvents.vue]:
```suggestion
        :to="`/events`"
```

File: frontend/components/LastExtrinsics.vue [hunk 3]
```diff
@@ -1,33 +1,29 @@
...
-          <p class="mb-0">
-            {{ data.item.section }} ➡
-            {{ data.item.method }}
-          </p>
-        </template>
-      </b-table>
+    <div class="headline">
+      <nuxt-link
+        v-b-tooltip.hover
+        :to="`/blocks`"
```
Comment (mariopino) [inline frontend/components/LastExtrinsics.vue]:
```suggestion
        :to="`/extrinsics`"
```

File: frontend/components/LastTransfers.vue [hunk 4]
```diff
@@ -1,65 +1,59 @@
...
-        <template #cell(amount)="data">
-          <p class="mb-0">
-            {{ formatAmount(data.item.amount) }}
-          </p>
-        </template>
-      </b-table>
+    <div class="headline">
+      <nuxt-link
+        v-b-tooltip.hover
+        :to="`/blocks`"
```
Comment (mariopino) [inline frontend/components/LastTransfers.vue]:
```suggestion
        :to="`/transfers`"
```

**Review by mariopino — CHANGES_REQUESTED** [review]

**Review by mariopino — APPROVED** [review]

**Pull request merged**